gears/flywheel/fsl-stats.json

```diff
@@ -0,0 +1,217 @@
+      "optional": true,
+      "type": "boolean"
+    },
+    "Entropy (nonzero)": {
+      "id": "E",
+      "description": "output mean entropy (of nonzero voxels)",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Volume": {
+      "id": "v",
+      "description": "output <voxels> <volume>",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Volume (nonzero)": {
+      "id": "V",
+      "description": "output <voxels> <volume> (for nonzero voxels)",
+      "optional": true,
+      "type": "boolean"
+    },
+    "ROI stats": {
+      "id": "w",
+      "description": "output smallest ROI <xmin> <xsize> <ymin> <ysize> <zmin> <zsize> <tmin> <tsize>\n containing nonzero voxels",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Max voxel coords": {
+      "id": "x",
+      "description": "output co-ordinates of maximum voxel",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Min voxel coords": {
+      "id": "X",
+      "description": "output co-ordinates of minimum voxel",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Center of gravity (mm)": {
+      "id": "c",
+      "description": "output centre-of-gravity (cog) in mm coordinates",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Center of gravity (voxels)": {
+      "id": "C",
+      "description": "output centre-of-gravity (cog) in voxel coordinates",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Absolute values?": {
+      "id": "a",
+      "description": "use absolute values of all image intensities",
+      "optional": true,
+      "type": "boolean"
+    },
+    "NaN/Inf as zero?": {
+      "id": "n",
+      "description": "treat NaN or Inf as zero for subsequent stats",
+      "optional": true,
+      "type": "boolean"
+    }
+  },
+  "custom": {
+    "docker-image": "flywheel/fsl-stats:0.1.5_5.0",
+    "gear-builder": {
+      "category": "analysis",
+      "image": "flywheel/fsl-stats:0.1.5_5.0"
+    },
+    "flywheel": {
+      "suite": "FSL"
+    }
+  },
+  "description": "This gear returns statistical output for a given NIFTI image.",
+  "environment": {
+    "FLYWHEEL": "/flywheel/v0"
+  },
+  "inputs": {
+    "input_image": {
+      "description": "Main image for which stats will be generated",
+      "base": "file",
+      "type": {
+        "enum": [
+          "nifti"
+        ]
+      }
+    },
+    "mask_image": {
+      "description": "(-k): use the specified image (filename) for masking - overrides lower and upper thresholds",
+      "optional": true,
+      "base": "file",
+      "type": {
+        "enum": [
+          "nifti"
+        ]
+      }
+    },
+    "difference_image": {
+      "description": "(-d): take the difference between the base image and the image specified here",
+      "optional": true,
+      "base": "file",
+      "type": {
+        "enum": [
+          "nifti"
+        ]
+      }
+    }
+  },
+  "label": "fsl-stats",
```

Comment:
Can we try to standardize how the label looks like among the FSL gear on gitlab. Fsl-fslhd goes as `FSL fslhd`.